arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -169,9 +177,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 }
 
 extension UIColor {
-    class var primaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.475, blue: 0.757, alpha: 1) }
-    class var secondaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.368, blue: 0.584, alpha: 1) }
-    class var backgroundGray: UIColor { return #colorLiteral(red: 0.973, green: 0.973, blue: 0.973, alpha: 1) }
-    class var primaryTextColor: UIColor { return #colorLiteral(red: 0.196, green: 0.196, blue: 0.196, alpha: 1) }
-    class var secondaryTextColor: UIColor { return #colorLiteral(red: 0.349, green: 0.349, blue: 0.349, alpha: 1) }
+    // Also used as global tint/accent color.
+    class var accentColor: UIColor { return UIColor(named: "AccentColor")! }
+    class var backgroundGray: UIColor { return UIColor(named: "backgroundGray")! }
+    class var backgroundGrayWithOpacity: UIColor { return UIColor(named: "backgroundGrayWithOpacity")! }
+    class var primaryTextColor: UIColor { return UIColor(named: "primaryTextColor")! }
```

Comment:
Yes. Since it is a black color in light mode, and doesn't make sense to change under dark mode as it is part of the map graphics, I change it to `UIColor.black` - `#000000`.